nixos/doc/manual/release-notes/rl-2411.section.md

```diff
@@ -309,6 +311,8 @@
 
 - The `services.trust-dns` module has been renamed to `services.hickory-dns`.
 
+- The legacy `immich-cli` package has been replaced with the new cli under 'immich.cli'.
```

Comment:
This means it will not be discoverable from s.n.o. anymore. Does it make sense to use the CLI from a remote machine to interact with Immich?

Reply:
I think so, as the cli can be used to upload images to a remote server.

Reply:
Correct - I currently rely on the CLI on my laptop to upload photos exported from my DSLR to Immich.